File: AtCoderBeginnersSelection/ABC081A/main.dart
import 'dart:io';

void main(List<String> args) {
  var num = 0;
  final masu = stdin.readLineSync()!;
  for (var i = 0; i < 3 && i < masu.length; i++) {
    if (masu[i] == '1') {
      num += 1;
    }
  }
  print(num);
}

File: AtCoderBeginnersSelection/ABC081A/main.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    int num = 0;
    string masu;
    cin >> masu;
    for (int i = 0; i < 3; i++)
    {
        if (masu.at(i) == '1')
        {
            num += 1;
        }
    }

    cout << num;
}
